### PR TITLE
A handful of Block mappings

### PR DESCRIPTION
**Pull request by gudenau**
Probably made a mess of things, first try after all.

### CODE REVIEW

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -113,6 +117,7 @@ CLASS bgs net/minecraft/block/Block
 		ARG 3 pos
 		ARG 4 state
 		ARG 5 blockEntity
+		ARG 6 itemStack
```
Comment (shadowfacts) [inline mappings/net/minecraft/block/Block.mapping]: `stack` is used as the parameter name for `ItemStack` parameters in `Item` methods. Should be the same here for consistency.

**Review by shadowfacts — CHANGES_REQUESTED** [review]
Local variables shouldn't be mapped until they're properly supported across all of Fabric's toolchain.

Other than that, looks good.

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (Prospector) [reply mappings/net/minecraft/block/Block.mapping]: agree with shadowfacts; this should be `stack`

**Review by Prospector — CHANGES_REQUESTED** [review]
Other than that one `itemStack` parameter, looks good 👍

**Comment by gudenau** [comment]
There are conflicts on this and I have no idea how one would fix that.

**Comment by gudenau** [comment]
I also don't know how to pull the suggested changes.

**Comment by shadowfacts** [comment]
Make the changes locally and then commit & push to your fork. I can deal with resolving the merge conflicts.

**Comment by gudenau** [comment]
Changed the itemStack param to stack.

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**